joint_trajectory_controller/CMakeLists.txt

```diff
@@ -48,17 +51,30 @@ add_library(${PROJECT_NAME} src/joint_trajectory_controller.cpp
                             include/trajectory_interface/quintic_spline_segment.h
                             include/trajectory_interface/pos_vel_acc_state.h)
 
-target_link_libraries(${PROJECT_NAME} ${catkin_LIBRARIES})
+target_link_libraries(${PROJECT_NAME} ${Boost_LIBRARIES} ${catkin_LIBRARIES})
 
 if(CATKIN_ENABLE_TESTING)
   find_package(catkin
     REQUIRED COMPONENTS
       actionlib
+      angles
+      cmake_modules
+      control_toolbox
+      control_msgs
+      controller_interface
       controller_manager
+      hardware_interface
+      pluginlib
+      realtime_tools
+      roscpp
+      rostest
+      rosunit
+      rqt_plot
+      trajectory_msgs
+      urdf
       xacro
   )
 
-  find_package(rostest REQUIRED)
```

Comment:
I would prefer to resolve `rostest` separately. (in all packages)

Reply:
Fixed at https://github.com/ros-controls/ros_controllers/pull/380/commits/e745c484517543b00b25111f5aa9dfe749c236ed